src/main/java/bit/javaoop/AbstractManager.java

```diff
@@ -0,0 +1,53 @@
+package bit.javaoop;
+import java.util.List;
+
+public abstract class AbstractManager extends AbstractEmployee {
+
+    private final Float budget = 100_000f;
+    private List <AbstractEmployee> EmployeeList = getEmployeeList();
+
+
+    public AbstractManager(String name, Float salary)
+    {
+       super(name,salary);
+    }
+
+
+    public abstract boolean canHire();
```

Comment:
I like what you did here. I think introducing this `canHire()` method is a great idea. However I think it would be more flexible if the method took `AbstractEmployee` as a parameter like:
`public abstract boolean canHire(AbstractEmployee employee)`